src/ng/parse.js

```diff
@@ -1882,6 +1881,8 @@ function $ParseProvider() {
  /**
   * @ngdoc method
   * @name $parseProvider#setIdentifierFns
+  * @this
```

Comment:
Ooc, why is this necessary? Doesn't ESLint allow methods of `@this` function to access `this`?